### DISCUSSION TITLE
Manual vulnerability scan already requested

### DISCUSSION THREAD

**Discussion by styxiner** [wazuh/wazuh#22849] [Q&A]
Yesterday I upgraded Wazuh on a docker container to v4.7.3 and a agent just to make some tests. [See docs](https://documentation.wazuh.com/current/deployment-options/docker/upgrading-wazuh-docker.html) and It didn't scan the updated agent, its last scan were the Apr 7, 2024 @ 21:22:53.000
Also, I'm trying to run a manual vulnerability scan via API and I don't how much time I will have to wait. Is there something I can do to watch a progress or I just wait??. If I try to do it throught the API console on wazuh dashboard It will show me a 404 Not found error

![Captura de pantalla 2024-04-09 114421](https://github.com/wazuh/wazuh/assets/160549592/3e072cc6-8dd2-4ec5-91b1-72bef21fc41b)

Also it didn't do any vulnerability check since the update

**Comment by MiguelazoDS** [comment]
Hi @Jaime-Moranchel, 

Sorry for the delay, I'm afraid that there's no possibility to track the progress of the scan once it is requested. Only a log like the following is the information we have that the requested scan was executed. 

```
2024/04/19 12:28:22 wazuh-modulesd:vulnerability-detector[38637] wm_vuln_detector.c:8664 at wm_vuldet_run_scan(): INFO: (5431): Starting vulnerability scan.
2024/04/19 12:28:22 wazuh-modulesd:vulnerability-detector[38637] wm_vuln_detector.c:2847 at wm_vuldet_check_agent_vulnerabilities(): INFO: (5450): Analyzing agent '000' vulnerabilities.
2024/04/19 12:28:22 wazuh-modulesd:vulnerability-detector[38637] wm_vuln_detector.c:2866 at wm_vuldet_check_agent_vulnerabilities(): INFO: (5471): Finished vulnerability assessment for agent '000'
```

If debug is enabled for logging you will get some extra logs, but not too much information besides the previous ones. 

Respecting to the execution through the API console please try using just "PUT /vulnerability"

**Comment by sebasfalcone** [comment]
Inactive thread, fell free to re open this issue
